app/components/search_result_caption_component.html.erb

```diff
@@ -1,10 +1,10 @@
 <span class="govuk-!-margin-right-4">
   <% if searched.present? && filters_count&.positive? %>
-    <strong><%= count %></strong> <%= item_label.pluralize(count) %> matching search and filters<br>
+    <strong><%= count %></strong> <%= item_label.pluralize(count) %> matching search and filters
```

Comment:
these were messing with the formatting, also for log downloads so I'm pretty sure they should have been removed already

Reply:
Hmm, I mean if you search and or filter logs it looks better with these new lines and I think it was in the design, cause otherwise the download links are split over several lines

Reply:
Ah I'd missed that - I've addressed this for schemes by adding nowrap to the download links, will do the same for the log link I think that's a better solution, rather than having an indented download link on a new line when there's loads of room above it. Sounds like the designs for log and scheme downloads contradict each other so I'll update both to the latest one.

Reply:
Have updated the user log download link too for consistency (even though I couldn't actually make it wrap incorrectly there anyway)